matlab/build_support/test.m

```diff
@@ -0,0 +1,28 @@
+function test()
+% Licensed to the Apache Software Foundation (ASF) under one
+% or more contributor license agreements.  See the NOTICE file
+% distributed with this work for additional information
+% regarding copyright ownership.  The ASF licenses this file
+% to you under the Apache License, Version 2.0 (the
+% "License"); you may not use this file except in compliance
+% with the License.  You may obtain a copy of the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing,
+% software distributed under the License is distributed on an
+% "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+% KIND, either express or implied.  See the License for the
+% specific language governing permissions and limitations
+% under the License.
+
+vars = common_vars();
+
+compile();
+
+originalPath = addpath(vars.srcDir, vars.buildDir);
+restoreOriginalPath = onCleanup(@()path(originalPath));
```

Comment:
Is there a styleguide for MATLAB library code?

Reply:
Somewhat surprisingly, there isn't an official MATLAB style guide.  The closest one I'm aware of is [MATLAB Style Guidelines 2.0](https://www.mathworks.com/matlabcentral/fileexchange/46056-matlab-style-guidelines-2-0).